### PR TITLE
Fix document error for ApplyFtrl, ApplyFtrlV2 and SparseApplyFtrlV2

### PR DESCRIPTION
**Pull request by blueyi**
The implementation of Ftrl optimizer has been updated, but the internal doc is incorrect.

### CODE REVIEW

File: tensorflow/core/api_def/base_api/api_def_SparseApplyFtrlV2.pbtxt [hunk 1]
```diff
@@ -72,8 +72,8 @@ END
   description: <<END
 That is for rows we have grad for, we update var, accum and linear as follows:
 grad_with_shrinkage = grad + 2 * l2_shrinkage * var
-accum_new = accum + grad_with_shrinkage * grad_with_shrinkage
-linear += grad_with_shrinkage +
+accum_new = accum + grad * grad
```
Comment (mihaimaruseac) [inline tensorflow/core/api_def/base_api/api_def_SparseApplyFtrlV2.pbtxt]: Same here

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Can you also point to the commit that changed code? Or the issue this fixes?

File: tensorflow/core/api_def/base_api/api_def_ApplyFtrlV2.pbtxt [hunk 2]
```diff
@@ -65,8 +65,8 @@ END
   summary: "Update \'*var\' according to the Ftrl-proximal scheme."
   description: <<END
 grad_with_shrinkage = grad + 2 * l2_shrinkage * var
-accum_new = accum + grad_with_shrinkage * grad_with_shrinkage
-linear += grad_with_shrinkage +
+accum_new = accum + grad * grad
```
[the remark this answers is not in the document]
Comment (blueyi) [reply tensorflow/core/api_def/base_api/api_def_ApplyFtrlV2.pbtxt]: Please reference this commit #https://github.com/tensorflow/tensorflow/commit/20d5683b826be03776978af3b8108fc3b5dc9cb8

File: tensorflow/core/api_def/base_api/api_def_SparseApplyFtrlV2.pbtxt (hunk 1 above)
Comment (blueyi) [reply tensorflow/core/api_def/base_api/api_def_SparseApplyFtrlV2.pbtxt]: Also reference here: #https://github.com/tensorflow/tensorflow/commit/20d5683b826be03776978af3b8108fc3b5dc9cb8

**Comment by gbaned** [comment]
@blueyi, @mihaimaruseac Any update on this PR? Please. Thanks!

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**